ConnectorRhino/ConnectorRhino/ConnectorRhinoShared/Utils.cs

```diff
@@ -370,7 +450,7 @@ public static string TimeAgo(string timestamp)
     {
       timeAgo = DateTime.Now.Subtract(DateTime.Parse(timestamp));
     }
-    catch (FormatException e)
+    catch (FormatException)
     {
       Debug.WriteLine("Could not parse the string to a DateTime");
       return "";
```

Comment:
Interesting that the format may be wrong on the timestamp. I think we'd want to log that as I can't imagine how that would happen, but since the catch is there, it must have been for a reason

Reply:
Actually just noticed there are no references to this method - going to remove it!